site/src/pages/release-notes/1.13.3.mdx

```diff
@@ -0,0 +1,53 @@
+---
+date: 2021-11-23
+---
+
+## 🌟 New features
+
+- N/A
+
+## 📈 Improvements
+
+- N/A
+
+## 🛠️ Bug fixes
+
+- <type://DecodingClient> does not raise an exception anymore when duplicated encodings are specified. #3932 #3934
+  - The duplicated encodings are silently ignored.
+- `unfinishedResponse` is correctly decremented when the <type://HttpResponse> is cancelled. #3920
```

Comment:
`unfinishedResponse` is an internal implementation term. Uses might not correctly understand the meaning. 
How about:
```
An early cancelled <type://HttpResponse> is now correctly removed from the active responses.
```